custodian/qchem/jobs.py

```diff
@@ -144,14 +143,28 @@ def opt_with_frequency_flattener(
         max_molecule_perturb_scale=0.3,
         check_connectivity=True,
         linked=True,
+        transition_state=False,
+        freq_before_opt=False,
         save_final_scratch=False,
         **QCJob_kwargs,
     ):
         """
         Optimize a structure and calculate vibrational frequencies to check if the
-        structure is in a true minima. If a frequency is negative, iteratively
-        perturbe the geometry, optimize, and recalculate frequencies until all are
-        positive, aka a true minima has been found.
+        structure is in a true minima.
+
+        If there are an inappropriate number of imaginary frequencies (0 for a
+         minimum-energy structure, 1 for a transition-state), attempt to re-calculate
+         using one of two methods:
```

Comment:
@espottesmith here, don't you mean `>0 for a minimum-energy structure or >1 for a transition state` ?

Reply:
Absolutely. Good catch.